app/controllers/workflow_executions/submission_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Workflow submission controller
+  class SubmissionController < ApplicationController
+    def pipeline_selection
+      respond_to do |format|
+        format.html { redirect_to dashboard_projects_path }
+        format.turbo_stream do
+          render 'workflow_executions/submissions/pipeline_selection'
```

Comment:
if you rename this controller to `submissions` instead of `submission` you won't have to specify the template to use as it will be auto selected.

Reply:
Thanks, updated in 689a4b924d24f9c33edc059cde63b94e1b5241ea